node/src/actors/chain_manager/actor.rs

```diff
@@ -47,6 +47,10 @@ impl Actor for ChainManager {
             .ok();
 
         self.secp = Some(CryptoEngine::new());
+
+        // Telemetry is configured here to avoid a race condition if configured directly
+        // upon actor start because of asynchronous nature of `get_key` function.
+        self.configure_telemetry_scope();
```

Comment:
Isn't the comment contradictory? This is the actor start

Reply:
Ooops I forgot to remove that function call from there. It belongs in `get_pkh`.